index.js

```diff
@@ -68,21 +69,28 @@ class Registrar extends Emitter {
 
   /**
    * Remove the registration for a user
-  * @param {String} aor - the address-of-record for the user
-  * @returns {Boolean} true if the registration was successfully removed
-  */
+   * @param {String} aor - the address-of-record for the user
+   * @returns {Boolean} true if the registration was successfully removed
+   */
   async remove(aor) {
     const key = makeUserKey(aor);
     try {
       const result = await this.client.del(key);
-      debug(`Registrar#remove ${aor} result: ${result}`);
+      const sortedSetResult = await this.client.zrem('active-user', aor);
```

Comment:
it looks to me like we added 'key' yet we are trying to remove 'aor', not 'key'.  Won't this fail?  If so, then we also need to add a test since this should have failed the test case